contracts/javascore/xcall/src/test/java/foundation/icon/btp/xcall/CallServiceTest.java

```diff
@@ -34,7 +34,6 @@ protected void teardown() {
 
 
     @Test
-    @Order(0)
```

Comment:
Please delete all @Order references from the test file.

Reply:
done